tests/pubsub_logging_test.py

```diff
@@ -41,6 +46,105 @@ def test_compat_urlsafe_b64encode(self):
         self.assertEqual(expected, result)
 
 
+class GetPubsubClientTest(unittest.TestCase):
```

Comment:
I would prefer not having these at all if they're not going to be done right. I don't think it would be too difficult to write a decent test for get_with_scopes, which is the more important of the two.